src/mini_file_server/core/handler.clj

```diff
@@ -32,10 +33,12 @@
       response
       (content-type "application/x-compressed")))
 
-(defroutes app-routes
-  (route/resources "/")
-  (GET "/" [] (index/render))
-  (GET "/list.json" [] (response (fs/files)))
+(defn authenticated? [id pass]
+  (and (= id "admin")
+       (= pass "admin")
```

Comment:
How about using an environment variable? e.g. `MFS_AUTH=admin:admin`

Reply:
Change the user to MFS_AUTH as your idea.